packages/components/organisms/f-checkout/src/components/ErrorDialog.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+    <mega-modal
+        v-if="errorCode"
+        :is-open="isOpen"
+        data-test-id="checkout-issue-modal"
+        has-overlay
+        @close="handleClose">
+        <h3 data-test-id="checkout-issue-modal-title" class="u-noSpacing">
+            {{ $t(`errorMessages.checkoutIssues.${errorCode}.title`) }}
+        </h3>
+
+        <p data-test-id="checkout-issue-modal-message">
+            {{ $t(`errorMessages.checkoutIssues.${errorCode}.message`) }}
+        </p>
+
+        <f-button
+            :class="$style['c-checkout-redirectButton']"
+            button-type="primary"
+            button-size="large"
+            action-type="button"
```

Comment:
I'm not sure you need `button-type="primary"` and `action-type="button"`. I think they're both defaults